OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -131,47 +499,221 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        of black and red grids, lines, and dots
+                                        (e.g. on <locus target="#30v #34v #37v"/>).</ab>
                                 </layout>
-                                
+                            </layoutDesc>
+                            
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h3" script="Ethiopic">
+                                     <locus from="29r" to="37v"/>
+                                <date notBefore="1800" notAfter="1949"/>
+                                <desc>Irregular and untrained hand, broad and rectangular script.
+                                The scribe often omits the word separator at the end of a line. It is possible that the hand is identical
+                                with <ref target="#h1"/>, though the script differs.</desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Incipits (double red lines, single words or groups of words); 
+                                    the word <foreign xml:lang="gez">ጸልዩ፡</foreign>;
+                                    elements of numerals.</seg>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e5">
+                                    <desc>Scribal corrections by the main hand are relatively rare. 
+                                        Occasionally, omitted words and characters are written interlineally (e.g. <locus target="#31v #33r"/>)
+                                        and erroneous characters are marked for deletion with surrounding dashes (e.g. <locus target="#32r #33v"/>).  </desc>
+                                </item>
+                            </list>
+                        </additions>
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                            <origDate notBefore="1800" notAfter="1949" evidence="lettering"/>
+                        </origin>
+                    </history>
+                    
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                  <!--  <listBibl type="secondary">
+                                        <bibl></bibl>
+                                    </listBibl>-->
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    </msPart>
+                    
+                                    <msPart xml:id="p3">
```

Comment:
This is indeed a tricky situation. Since codicological and textual boundiers overlap, we need to decide whether to give priority in the description to the material boundaries or to the textual ones. In my understanding p1, p2, and p3 are identified by material boundaries, so I perfectly agree that they correspond to fols 1-28, 29-37, and 38-41. Also, the strcucture of encourages us to give priority to p over the textual description. What might be unclear to the reader is why textual material which is actually found on p2 (e.g. `<locus target="#26v"/><locus from="27r" to="27v"/>`) is described within p3.
I am just wondering (I do not know the established workflow in these cases, even though I think that there are probably similar cases of overlapping in the BM database) if text found on fols belonging to p2 should be nested within p2 and text on fols belonging to p3 within p3, with some clarifications such as "the text is the continuation of ... it is written in the same hand of ..." etc. Do you think this is a too rigid interpretation of the hierarchical levels of description?

Reply:
Thank you, yes, it might need more discussion. I write my thoughts down here, then we can see whether we need a meeting for this. 
The underlying question is what we understand by "msPart"/"Codicological Unit", our main unit of description: unit of material, unit of production, ...? The BM approach is that of a "unit of production", with addition of both content and material. So for instance, if a manuscript contains later glosses within the main text body, this is an addition, not a msPart, because while there is an addition of content, there is no addition of material. 
What we have here is addition of content on existing units of material and addition of content on new material within one unit of production (transformation model A3 according to Syntaxe to Codex). If we take msPart to mean unit of production, I think my division makes sense. If we take a more strict material approach and intend msPart as unit of material (which will maybe be more intuitive to readers), then p3 would only encompass 38-41, while the text on 26v, 27 etc. would be encoded as an addition in p2.
Regardless of the encoding we choose, the transformation A3 can be expressed precisely using the SdC relations (I didn't do it yet, because we won't be able to render it in print), but it will be good to include this once we have agreed on the encoding (so that the xml:ids don't have to be corrected a thousand times).

Reply:
Thank you Thea, now your encoding is much clearer to me. You are right: the underlying question is how we define msPart and "unit of production". Let's see how this record is visualized in the PDF output and possibly adjust the description with some comments if we strongly feel that the readers might be confused somehow. Also, a  simple reference to the transformation model A3 from Syntaxe to Codex might be useful (perhaps in the shape of a footnote).